<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ElasticConstatntsWidget</class>
 <widget class="QWidget" name="ElasticConstatntsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QGroupBox" name="materialGroupBox">
     <property name="title">
      <string>Материал</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="isIsotropy">
        <property name="text">
         <string>Изотропия</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="isOrthotropyENuG">
        <property name="text">
         <string>Ортотропия (три независимых параметра)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="isOrthotropy">
        <property name="text">
         <string>Ортотропия (общий случай)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_13">
     <item>
      <widget class="QGroupBox" name="elasticModulusGroupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Модулю Юнга</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="eLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>E=</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="e">
            <property name="decimals">
             <number>5</number>
            </property>
            <property name="maximum">
             <double>1000000000.000000000000000</double>
            </property>
            <property name="value">
             <double>203200.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="e1Label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;E&lt;span style=&quot; vertical-align:sub;&quot;&gt;1&lt;/span&gt;=&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="e1">
            <property name="decimals">
             <number>5</number>
            </property>
            <property name="maximum">
             <double>1000000000.000000000000000</double>
            </property>
            <property name="value">
             <double>203200.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="e2Label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;E&lt;span style=&quot; vertical-align:sub;&quot;&gt;2&lt;/span&gt;=&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="e2">
            <property name="decimals">
             <number>5</number>
            </property>
            <property name="maximum">
             <double>1000000000.000000000000000</double>
            </property>
            <property name="value">
             <double>203200.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="e3Label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;E&lt;span style=&quot; vertical-align:sub;&quot;&gt;3&lt;/span&gt;=&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="e3">
            <property name="decimals">
             <number>5</number>
            </property>
            <property name="maximum">
             <double>1000000000.000000000000000</double>
            </property>
            <property name="value">
             <double>203200.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="poissonGroupBox">
       <property name="title">
        <string>Коэффициент Пуассона</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="nuLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Liberation Serif, serif'; font-size:12pt;&quot;&gt;ν=&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="nu">
            <property name="decimals">
             <number>5</number>
            </property>
            <property name="maximum">
             <double>1.000000000000000</double>
            </property>
            <property name="value">
             <double>0.270000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="nu12Label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Liberation Serif, serif'; font-size:12pt;&quot;&gt;ν&lt;/span&gt;&lt;span style=&quot; font-family:'Liberation Serif, serif'; font-size:12pt; vertical-align:sub;&quot;&gt;12&lt;/span&gt;&lt;span style=&quot; font-family:'Liberation Serif, serif'; font-size:12pt;&quot;&gt;=&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="nu12">
            <property name="decimals">
             <number>5</number>
            </property>
            <property name="maximum">
             <double>1.000000000000000</double>
            </property>
            <property name="value">
             <double>0.270000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="nu13Label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Liberation Serif, serif'; font-size:12pt;&quot;&gt;ν&lt;/span&gt;&lt;span style=&quot; font-family:'Liberation Serif, serif'; font-size:12pt; vertical-align:sub;&quot;&gt;13&lt;/span&gt;&lt;span style=&quot; font-family:'Liberation Serif, serif'; font-size:12pt;&quot;&gt;=&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="nu13">
            <property name="decimals">
             <number>5</number>
            </property>
            <property name="maximum">
             <double>1.000000000000000</double>
            </property>
            <property name="value">
             <double>0.270000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="nu23Label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Liberation Serif, serif'; font-size:12pt;&quot;&gt;ν&lt;/span&gt;&lt;span style=&quot; font-family:'Liberation Serif, serif'; font-size:12pt; vertical-align:sub;&quot;&gt;23&lt;/span&gt;&lt;span style=&quot; font-family:'Liberation Serif, serif'; font-size:12pt;&quot;&gt;=&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="nu23">
            <property name="decimals">
             <number>5</number>
            </property>
            <property name="maximum">
             <double>1.000000000000000</double>
            </property>
            <property name="value">
             <double>0.270000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="shearGroupBox">
       <property name="title">
        <string>Модуль сдвига</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLabel" name="gLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Liberation Serif, serif'; font-size:12pt;&quot;&gt;G=&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="g">
            <property name="decimals">
             <number>5</number>
            </property>
            <property name="maximum">
             <double>1000000000.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>1000000000.000000000000000</double>
            </property>
            <property name="value">
             <double>80000.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QLabel" name="g12Label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Liberation Serif, serif'; font-size:12pt;&quot;&gt;G&lt;/span&gt;&lt;span style=&quot; font-family:'Liberation Serif, serif'; font-size:12pt; vertical-align:sub;&quot;&gt;12&lt;/span&gt;&lt;span style=&quot; font-family:'Liberation Serif, serif'; font-size:12pt;&quot;&gt;=&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="g12">
            <property name="decimals">
             <number>5</number>
            </property>
            <property name="maximum">
             <double>1000000000.000000000000000</double>
            </property>
            <property name="value">
             <double>80000.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QLabel" name="g13Label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Liberation Serif, serif'; font-size:12pt;&quot;&gt;G&lt;/span&gt;&lt;span style=&quot; font-family:'Liberation Serif, serif'; font-size:12pt; vertical-align:sub;&quot;&gt;13&lt;/span&gt;&lt;span style=&quot; font-family:'Liberation Serif, serif'; font-size:12pt;&quot;&gt;=&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="g13">
            <property name="decimals">
             <number>5</number>
            </property>
            <property name="maximum">
             <double>1000000000.000000000000000</double>
            </property>
            <property name="value">
             <double>80000.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <item>
           <widget class="QLabel" name="g23Label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Liberation Serif, serif'; font-size:12pt;&quot;&gt;G&lt;/span&gt;&lt;span style=&quot; font-family:'Liberation Serif, serif'; font-size:12pt; vertical-align:sub;&quot;&gt;23&lt;/span&gt;&lt;span style=&quot; font-family:'Liberation Serif, serif'; font-size:12pt;&quot;&gt;=&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="g23">
            <property name="decimals">
             <number>5</number>
            </property>
            <property name="maximum">
             <double>1000000000.000000000000000</double>
            </property>
            <property name="value">
             <double>80000.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>isOrthotropy</sender>
   <signal>toggled(bool)</signal>
   <receiver>eLabel</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>66</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>36</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>isOrthotropy</sender>
   <signal>toggled(bool)</signal>
   <receiver>e</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>167</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel">
     <x>167</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>isOrthotropy</sender>
   <signal>toggled(bool)</signal>
   <receiver>e1Label</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>45</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>isOrthotropy</sender>
   <signal>toggled(bool)</signal>
   <receiver>e1</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>173</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>167</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>isOrthotropy</sender>
   <signal>toggled(bool)</signal>
   <receiver>e2Label</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>97</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>45</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>isOrthotropy</sender>
   <signal>toggled(bool)</signal>
   <receiver>e2</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>125</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>167</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>isOrthotropy</sender>
   <signal>toggled(bool)</signal>
   <receiver>e3Label</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>164</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>45</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>isOrthotropy</sender>
   <signal>toggled(bool)</signal>
   <receiver>e3</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>215</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>167</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>isOrthotropy</sender>
   <signal>toggled(bool)</signal>
   <receiver>nuLabel</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>126</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>227</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>isOrthotropy</sender>
   <signal>toggled(bool)</signal>
   <receiver>nu</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>281</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>313</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>isOrthotropy</sender>
   <signal>toggled(bool)</signal>
   <receiver>nu12Label</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>229</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>isOrthotropy</sender>
   <signal>toggled(bool)</signal>
   <receiver>nu12</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>isOrthotropy</sender>
   <signal>toggled(bool)</signal>
   <receiver>nu13Label</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>110</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>231</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>isOrthotropy</sender>
   <signal>toggled(bool)</signal>
   <receiver>nu13</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>42</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>273</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>isOrthotropy</sender>
   <signal>toggled(bool)</signal>
   <receiver>nu23Label</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>18</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>231</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>isOrthotropy</sender>
   <signal>toggled(bool)</signal>
   <receiver>nu23</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>271</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>isIsotropy</sender>
   <signal>toggled(bool)</signal>
   <receiver>shearGroupBox</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>411</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>isOrthotropy</sender>
   <signal>toggled(bool)</signal>
   <receiver>gLabel</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>288</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel">
     <x>437</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>isOrthotropy</sender>
   <signal>toggled(bool)</signal>
   <receiver>g</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>281</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel">
     <x>482</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>isOrthotropy</sender>
   <signal>toggled(bool)</signal>
   <receiver>g12Label</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>308</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>444</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>isOrthotropy</sender>
   <signal>toggled(bool)</signal>
   <receiver>g12</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>489</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>isOrthotropy</sender>
   <signal>toggled(bool)</signal>
   <receiver>g13Label</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>442</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>isOrthotropy</sender>
   <signal>toggled(bool)</signal>
   <receiver>g13</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>183</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>486</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>isOrthotropy</sender>
   <signal>toggled(bool)</signal>
   <receiver>g23Label</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>201</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>isOrthotropy</sender>
   <signal>toggled(bool)</signal>
   <receiver>g23</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>521</x>
     <y>289</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
